src/pybullet_helpers/robots/__init__.py

```diff
@@ -1,16 +1,25 @@
 """Handles the creation of robots."""
-from typing import Sequence
+from typing import Dict
 
+from predicators.src.pybullet_helpers.geometry import Pose, Pose3D, Quaternion
+from predicators.src.pybullet_helpers.robots.fetch import FetchPyBulletRobot
 from predicators.src.pybullet_helpers.robots.single_arm import \
-    FetchPyBulletRobot, SingleArmPyBulletRobot
-from predicators.src.structs import Pose3D
+    SingleArmPyBulletRobot
+
+# Note: these are static base poses which suffice for the current environments.
+_ROBOT_TO_BASE_POSE: Dict[str, Pose] = {
+    "fetch": Pose(position=(0.75, 0.7441, 0.0)),
+}
 
 
 def create_single_arm_pybullet_robot(
-        robot_name: str, ee_home_pose: Pose3D, ee_orientation: Sequence[float],
+        robot_name: str, ee_home_pose: Pose3D, ee_orientation: Quaternion,
         physics_client_id: int) -> SingleArmPyBulletRobot:
     """Create a single-arm PyBullet robot."""
     if robot_name == "fetch":
-        return FetchPyBulletRobot(ee_home_pose, ee_orientation,
-                                  physics_client_id)
+        base_pose = _ROBOT_TO_BASE_POSE.get(robot_name, Pose.identity())
```

Comment:
do we really want to have a default here? shouldn't all robots specify a base pose?

Reply:
Yes at least for now let's constrain it to a specified one. Will update